website/static/js/project.js

```diff
@@ -313,6 +313,10 @@ $(document).ready(function() {
         window.location = '/search/?q=(tags:"' + $(e.target).text().toString().trim()+ '")';
     });
 
+    $('#shareDropDownMenu li a').on('click', function(e) {
+        window.open(this.href, '', 'menubar=no,toolbar=no,resizable=yes,scrollbars=yes,width=600,height=400');
```

Comment:
It's generally considered bad UI practice to force something to open in a new window -- has this been approved by the product team?